src/interpreter/naive_interpreter.nit

```diff
@@ -314,6 +314,15 @@ class NaiveInterpreter
 		return frames.first.arguments.first.mtype.as(MClassType)
 	end
 
+	# Initialize the environment for a call and return a new Frame
```

Comment:
I think that *environment* is a bad word because I do not understand what you mean. I suggest that you use precise terms.